server/views/pages/goal/archive/reason.njk

```diff
@@ -0,0 +1,91 @@
+{% extends "../../../partials/layout.njk" %}
+
+{% set pageId = "archive-goal" %}
+{% set pageTitle = "You are archiving a goal for the prisoner" %}
+
+{% block beforeContent %}
+  {% include "../../../partials/breadcrumb.njk" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">Select a reason for archiving this goal</h1>
+      <section class="archive-goal-summary">
```

Comment:
I guess there is not a standard GDS component for this grey banner 🤷‍♂️ 

This is one of the areas where we need to be careful, as the designers just "design stuff" and it isn't always fully GDS compliant (not to say they should only ever use GDS components, more we've learnt its usually a red flag when they design stuff that does not have a standard component)

(This is more of an FYI than a comment or request to change 👍 )

Reply:
Yeah, I tried to use the semantic markup and GDS classes so hopefully it's not too far away from something GDS compliant but looks like figma